template/acc_logs.php

```diff
@@ -10,91 +10,9 @@
 		if(file_exists($file_to_delete))
 			unlink($file_to_delete);
 	}
-
```

Comment:
The below code is to get rid of obsolete settings.  Hopefully I cut just the right amount of code.